src/coreclr/jit/emitxarch.cpp

```diff
@@ -10657,6 +10708,37 @@ void emitter::emitDispEmbBroadcastCount(instrDesc* id)
     printf(" {1to%d}", vectorSize / baseSize);
 }
 
+// emitDispEmbRounding: Display the tag where embedded rounding is activated
+//
+// Arguments:
+//   id - The instruction descriptor
+//
+void emitter::emitDispEmbRounding(instrDesc* id)
+{
+    if (!id->idIsEvexbContext())
+    {
+        return;
+    }
+    assert(!id->idHasMem());
+    unsigned roundingMode = id->idGetEvexbContext();
+    if (roundingMode == 1)
+    {
+        printf(" {rd-sae}");
```

Comment:
nit: in the Intel manual I have, section "15.6.4 Static Rounding Mode and Suppress All Exceptions", there are these examples:
```
vaddps zmm7 {k6}, zmm2, zmm4, {rd-sae}
vmulps zmm7 {k6}, zmm2,[rax], {rd-sae}
vaddps ymm7 {k6}, ymm2, ymm4,{rd-sae}
```

In each case there is a comma before the `{rd-sae}`.

It's not clear from the instruction set reference whether the comma is expected or not. So what is the standard? Comma or no comma?

Reply:
Sorry commented in the wrong place, please see [comment](https://github.com/dotnet/runtime/pull/94684#discussion_r1456669338)